jquery.highchartTable.js

```diff
@@ -315,7 +315,8 @@
       var lineShadow  = $table.data('graph-line-shadow');
       var lineWidth   = $table.data('graph-line-width') || 2;
 
-      for(var i=0; i<9; i++) {
+      var nbOfColors = Math.max(defaultColors.length, themeColors.length);
+      for(var i=0; i < nbOfColors; i++) {
```

Comment:
Crockford il dit « JavaScript does not have block scope, so defining variables in blocks can confuse programmers who are experienced with other C family languages. Define all variables at the top of the function. ».

Bon, il dit aussi des trucs avec lesquels je ne suis pas d'accord, mais sur ce point, je trouve qu'il a raison.